server/controllers/article.controller.js

```diff
@@ -149,22 +149,31 @@ class ArticleController {
      * @returns {object}
      * @memberof getAllArticles
     */
+    const { page } = req.params;
+    const limit = 10;
+    const offset = limit * (page - 1);
+    const order = [['updatedAt', 'DESC']];
     const articles = await Article.findAndCountAll({
+      limit,
+      offset,
+      order,
       where: { published: true },
```

Comment:
Good work on checking for the published state

Reply:
Yeah...very true!!!
Nice work